index.js

```diff
@@ -28,6 +29,7 @@
     }
 });
 
+app.use(apiTokenMiddleware);
```

Comment:
@blokhin @knopki
Should we use a rate limiting library? https://codeql.github.com/codeql-query-help/javascript/js-missing-rate-limiting/

Reply:
@pyoner yes, sure (it seems easy to add)

Reply:
Create an issue about rate limiting for all request (not only authorization request) and ignore this warning :)

Reply:
btw, why middleware is registered here and not in `bff` config? It is not easy to guess order of middlewares.

Reply:
@knopki 
> btw, why middleware is registered here and not in `bff` config? It is not easy to guess order of middlewares.

because `bff` middlewares are registered after `sse`

Reply:
> @pyoner yes, sure (it seems easy to add)

@blokhin but on the production server we are using a `nginx` proxy server, which can rate limit via config

Reply:
Rate-limiting at the application side is favourable in addition to the proxy side since we might differentiate endpoints and authenticated users.